packages/web/src/components/withdraw-usdc-modal/components/PrepareTransfer.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect } from 'react'
+
+import {
+  CoinflowWithdrawalState,
+  withdrawUSDCSelectors,
+  WithdrawUSDCModalPages,
+  useWithdrawUSDCModal
+} from '@audius/common'
+import { Flex, Text } from '@audius/harmony'
+import { useSelector } from 'react-redux'
+
+import LoadingSpinner from 'components/loading-spinner/LoadingSpinner'
+
+import styles from './PrepareTransfer.module.css'
+
+const { getCoinflowState } = withdrawUSDCSelectors
+
+const messages = {
+  holdOn: 'Hold on!',
+  preparingTransfer:
+    "We're preparing your transfer. This could take a few moments."
```

Comment:
Can we add a "Please don't close your browser" or something similar?

Reply:
Yep, can do.